### PR TITLE
Set up Single Table Inheritance using Devise for Admin, Buyer, Broker

### PR DESCRIPTION
**Pull request by jerickcamacho**
The Story: 
You can create Admin, Buyer and Broker User then you can sign in your created User, we also have TDD Test for the users to specify their type as user. 

Screenshots: 
admin login
Admin Sign [)
Broker Sign in
Broker Sign Up
Buyer Sign In
Buyer Sign Up
Sign in Admin

Reference: https://sodocumentation.net/ruby-on-rails/topic/9125/single-table-inheritance

### CODE REVIEW

**Review by equeferrer — APPROVED** [review]
Good job, Jerick!

File: config/routes.rb [hunk 1]
```diff
@@ -1,5 +1,11 @@
 Rails.application.routes.draw do
-  devise_for :users
-  # root "home#index"
+  get 'home/index'
+  #devise_for :users
```
Comment (analyn-cajocson) [inline config/routes.rb]: remove this unnecessary comment

Comment (jerickcamacho) [reply config/routes.rb]: This is noted Ma'am, next time, I will remove unnecessary comment, thank you for reminding me

Comment (analyn-cajocson) [reply config/routes.rb]: You have to remove the unnecessary comments first before I approve your PR

**Pull request merged**